core/src/main/java/com/afollestad/materialdialogs/internal/list/MultiChoiceDialogAdapter.kt

```diff
@@ -134,7 +135,7 @@ internal class MultiChoiceDialogAdapter(
   }
 
   override fun positiveButtonClicked() {
-    if (currentSelection.isNotEmpty()) {
+    if (allowEmptySelection or currentSelection.isNotEmpty()) {
```

Comment:
Is there a reason you're using `or` instead of `||` here? They aren't equal in what they do, exactly

Reply:
Did not know this. So if I use or instead of || corrected ordering does not have any advantage, good to know